base/src/notification-manager/notification.ts

```diff
@@ -9,20 +9,26 @@
   Copyright Contributors to the Zowe Project.
 */
 
-
+const defaultStyleClass = "org_zowe_zlux_ng2desktop_snackbar";
 export class ZoweNotification {
   private message: string;
   private date: Date;
   private type: MVDHosting.ZoweNotificationType;
   private plugin: string;
   private title: string;
+  private styleClass?: string;
 
-  constructor(title: string, message: string, type: MVDHosting.ZoweNotificationType, plugin: string) {
+  constructor(title: string, message: string, type: MVDHosting.ZoweNotificationType, plugin: string, styleClass?: string) {
     this.title = title;
     this.message = message;
     this.type = type;
     this.date = new Date();
     this.plugin = plugin;
+    if (this.styleClass) {
```

Comment:
No `this`
Actually, there is a typescript way to do this entire thing if you want. I believe it would be:
`constructor(title: string, message: string, type: MVDHosting.ZoweNotificationType, plugin: string, private styleClass: string=defaultStyleClass) `